Formalist/AccessoryViewToolbar.swift

```diff
@@ -21,19 +21,24 @@ class AccessoryViewToolbar: UIToolbar {
         return UIBarButtonItem(title: "Next", style: .plain, target: self, action: #selector(nextAction))
     }()
 
-    lazy var doneButtonItem: UIBarButtonItem = {
-        return UIBarButtonItem(barButtonSystemItem: .done, target: self, action: #selector(done))
-    }()
+    private func doneButtonItem(customTitled title:String? = nil) -> UIBarButtonItem {
+        if let title = title {
+            return UIBarButtonItem(title: title, style: .plain, target: self, action: #selector(done))
+        } else {
+
+            return UIBarButtonItem(barButtonSystemItem: .done, target: self, action: #selector(done))
+        }
+    }
 
     var callbacks: AccessoryViewToolbarCallbacks?
 
-    override init(frame: CGRect) {
+    init(frame: CGRect, doneButtonCustomTitle: String? = nil) {
```

Comment:
Considering this is no longer just Done, did you consider having `init(accessoryButton: UIBarButtonItem)` for the most flexibility?

Reply:
We could definitely add even more flexibility here – but the implementation associates the user-defined `AccessoryViewToolbarCallbacks.doneAction` with the button on the right of the toolbar, so it'd take more thoughtful renaming of these buttons & their purposes.

Do you think it's worth the time to do that?

Reply:
Probably not. This just feels like we're bolting something very specific to our use-case onto the framework.

Reply:
Definitely. I'll make a task to allow more general customization of these buttons, so we can make a stronger link between the `action`s and the button titles; it felt a little rabbit-hole-like to really refactor that framework component to support this title change, but you're right, we shouldn't make a habit of shoe-horning like this!

Reply:
Task is here: https://github.com/seedco/Formalist/issues/66